Modules/module2/04-core-dns.md

```
@@ -0,0 +1,346 @@
+kubectl logs -n kube-system <coredns-pod> --previous
+
+# 3. Describe pod
+kubectl describe pod -n kube-system <coredns-pod>
+
+# 4. Check events
+kubectl get events -n kube-system --sort-by='.lastTimestamp'
+```
+
+**Common Causes and Solutions**:
+
+**A. Corefile syntax error**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)